Vagrantfile

```diff
@@ -44,8 +44,7 @@ Vagrant.configure(2) do |config|
   config.vm.synced_folder "../task-runner", "/data/www/task-runner", owner: "vagrant", group: "www-data"
   config.vm.synced_folder "../website", "/data/www/website", owner: "vagrant", group: "www-data"
   config.vm.synced_folder "../log", "/var/log/permanent", owner: "vagrant", group: "www-data", mount_options: ["dmode=770", "fmode=660"]
-  config.vm.synced_folder "../uploader", "/data/www/uploader", owner: "vagrant", group: "www-data"
-  config.vm.synced_folder "../mdot", "/data/www/mdot", owner: "vagrant", group: "www-data"
+  config.vm.synced_folder "../web-app", "/data/www/mdot", owner: "vagrant", group: "www-data"
```

Comment:
This is so great; I was just thinking about how to make my local copy use web-app instead of mdot.  Thanks!

Reply:
+1 here, meant to make a PR for this already but forgot!

Reply:
One other thing though - in deploy.sh we run node install for all the mdot packages and move to `node_modules` inside mdot.  I expect we should just do the same thing for web-app.  Is that what you think, @andrewatwood ?

Reply:
(Lines 62-70, I'm talking about)

Reply:
Are you saying that's not working for you, @cecilia-donnelly ? From reading it (but not running it), I expected it to continue to work, since it's the same code mounted in the same place.

More broadly, I think this question is related to #12. I've been building mdot/web-app on my host rather than my guest, so as a workaround, I've deleted those lines (deploy.sh 62-70) locally in order to speed up `vagrant up`. I haven't discussed that enough with the team to figure out if that's worth opening a pull request for; other things have been higher priority.

Reply:
No, it worked okay!  The real question is probably more complicated: where else do we depend on mdot being named mdot?  If the new repo is called web-app then I'd rather refer to the client that way everywhere, for simplicity's sake, but I can see that changing that might be a bigger project.